app/controllers/simulator_controller.rb

```diff
@@ -22,6 +23,9 @@ def embed
       render plain: "Project has been moved or deleted. If you are the owner of the project, Please check your project access privileges." and return
     end
     @logix_project_id = params[:id]
+    @project = Project.find(params[:id])
+    @author = @project.author_id
+    @show_value = 1
```

Comment:
This is the correct logic but I would prefer the variable to be a boolean instead of integer and the name to be @external_embed = true/false

Reply:
cool